docker-compose.yml

```diff
@@ -168,7 +168,7 @@ services:
   nginx:
     << : *restart_policy
     ports:
-      - '9000:80/tcp'
+      - '$SENTRY_ONPREMISE_PORT:80/tcp'
```

Comment:
As a nice side effect, this would also allow restricting the port forwarding to `127.0.0.1`:

```
SENTRY_ONPREMISE_PORT=127.0.0.1:9000
```

Reply:
TIL you can use IP/Host here, nice! (ref to the docs where this is mentioned: https://docs.docker.com/compose/compose-file/#ports)